packages/matter.js/src/CommissioningServer.ts

```diff
@@ -274,24 +267,27 @@ export class CommissioningServer extends MatterNode {
 
         // Set the required basicInformation and respect the provided values
         // TODO Get the defaults from the cluster meta details
-        const basicInformationAttributes = Object.assign(
-            {
-                dataModelRevision: MATTER_DATAMODEL_VERSION,
-                nodeLabel: "",
-                hardwareVersion: 0,
-                hardwareVersionString: "0",
-                location: "XX",
-                localConfigDisabled: false,
-                softwareVersion: 1,
-                softwareVersionString: "v1",
-                capabilityMinima: {
-                    caseSessionsPerFabric: 3, // TODO get that limit from Sessionmanager or such or sync with it, add limit? Just a minima?
-                    subscriptionsPerFabric: 3, // TODO get that limit from Interactionserver? Respect it? It is just a minima?
-                },
-                serialNumber: `node-matter-${Crypto.get().getRandomData(4).toHex()}`,
+        const basicInformationAttributes: AttributeInitialValues<typeof BasicInformationCluster.attributes> = {
+            dataModelRevision: Specification.DATA_MODEL_REVISION,
+            nodeLabel: "",
+            hardwareVersion: 0,
+            hardwareVersionString: "0",
+            location: "XX",
+            localConfigDisabled: false,
+            softwareVersion: 1,
+            softwareVersionString: "v1",
+            capabilityMinima: {
+                caseSessionsPerFabric: 3, // TODO get that limit from Sessionmanager or such or sync with it, add limit? Just a minima?
+                subscriptionsPerFabric: 3, // TODO get that limit from Interactionserver? Respect it? It is just a minima?
             },
-            options.basicInformation,
-        ) as AttributeInitialValues<typeof BasicInformationCluster.attributes>;
+            serialNumber: `node-matter-${Crypto.get().getRandomData(4).toHex()}`,
```

Comment:
officially out of scope ... but should we change that to matter.js :-))

Reply:
will do